js/library.js

```diff
@@ -137,8 +87,7 @@ function createPopupProBlocker() {
 
         $('.visualizer-chart-shortcode').click(function (e) {
 
-            if ( ! visualizer.is_pro_user && e.target.classList.contains('viz-is-pro-chart') ) {
-                createPopupProBlocker();
+            if ( createPopupProBlocker( $, e ) ) {
```

Comment:
A small NIT here, if you want we can also rename the `e` to `event` to be consistent with the rest of the naming further down.